content/en/tracing/trace_collection/library_config/cpp.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Configuring the C++ Tracing Library
+kind: documentation
+code_lang: cpp
+type: multi-code-lang
+code_lang_weight: 50
+further_reading:
+- link: "https://github.com/DataDog/dd-opentracing-cpp"
+  tag: "Github"
+  text: Source code
+- link: "/tracing/visualization/"
+  tag: "Documentation"
+  text: "Explore your services, resources and traces"
+---
+
+After you set up the tracing library with your code and configure the Agent to collect APM data, optionally configure the tracing library as desired, including setting up [Unified Service Tagging][1].
+## Environment variables
+
+`DD_AGENT_HOST` 
+: **Version**: v0.3.6 <br>
+**Default**: `localhost` <br>
+Sets the host where traces are sent (the host running the Agent). Can be a hostname or an IP address. Ignored if `DD_TRACE_AGENT_URL` is set.
+
+`DD_TRACE_AGENT_PORT` 
+: **Version**: v0.3.6 <br>
+**Default**: `8126` <br>
+Sets the port where traces are sent (the port where the Agent is listening for connections). Ignored if `DD_TRACE_AGENT_URL` is set.
+
+`DD_TRACE_AGENT_URL` 
+: **Version**: v1.1.4 <br>
+Sets the URL endpoint where traces are sent. Overrides `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` if set. This URL supports http, https and unix address schemes.
+
+`DD_ENV` 
+: **Version**: v1.0.0 <br>
+If specified, adds the `env` tag with the specified value to all generated spans.
+
+`DD_SERVICE` 
+: **Version**: v1.1.4 <br>
+If specified, sets the default service name. Otherwise the service name must be provided via TracerOptions or JSON configuration.
```

Comment:
```suggestion
If specified, sets the default service name. Otherwise, the service name must be provided via TracerOptions or JSON configuration.
```